tracer/src/Datadog.Trace/Ci/Tags/TestSuiteVisibilityTags.cs

```diff
@@ -32,4 +32,9 @@ internal static class TestSuiteVisibilityTags
     /// Test session working directory environment variable
     /// </summary>
     public const string TestSessionWorkingDirectoryEnvironmentVariable = "DD_TESTSESSION_WORKINGDIRECTORY";
+
+    /// <summary>
+    /// Test session auto injected environment variable
+    /// </summary>
+    public const string TestSessionAutoInjectedEnvironmentVariable = "DD_CIVISIBILITY_AUTO_INSTRUMENTATION_PROVIDER";
```

Comment:
Need a dd-go PR for this setting? I _guess_ not, but then _should_ they be able to set it through other config sources (i.e. _should_ it be "real" config) and if not, you probably need to add it to the "exclude" list in the unit tests?

Reply:
Because this is a setting that we want to use internally and not expose it to customers I don't think we want to add it to the dd-go settings. This settings will be set by a bash script and only using environment variables.